be/src/common/config.h

```diff
@@ -835,6 +835,9 @@ CONF_Int64(deliver_broadcast_rf_passthrough_bytes_limit, "131072");
 // in passthrough style, the number of inflight RPCs of parallel deliveries are issued is not exceeds this limit.
 CONF_Int64(deliver_broadcast_rf_passthrough_inflight_num, "10");
 CONF_Int64(send_rpc_runtime_filter_timeout_ms, "1000");
+// if runtime filter size is larger than send_runtime_filter_via_http_rpc_min_size, be will transmit runtime filter via http protocol.
+// this is a default value, maybe changed by global_runtime_filter_rpc_http_min_size in session variable.
+CONF_Int64(send_runtime_filter_via_http_rpc_min_size, "67108864");
```

Comment:
640MB?

Reply:
> 640MB?

640MB is too large. I found that it will cost hundreds of milliseconds in server to receive a ~500+MB rf during my test. so I use 64MB here, just keep the same as the default value of rpc_http_min_size in exchange.